apps/oxlint/src/output_formatter/default.rs

```diff
@@ -198,21 +227,4 @@ mod test {
             "\nFound 6 warnings and 4 errors.\nExceeded maximum number of warnings. Found 6.\n"
         );
     }
-
-    #[test]
-    fn reporter_error() {
```

Comment:
removed the test because the tester is using it already. or else I need to access `DefaultOutputFormatter.get_diagnostic_reporter()` to get the test reporter, which feels a bit wierd.